src/gcp/cloudsql/interactive.ts

```diff
@@ -0,0 +1,55 @@
+import * as pg from "pg";
+import * as ora from "ora";
+import chalk from "chalk";
```

Comment:
Please use `colorette` instead of `chalk` - we don't currently have a direct dependency on chalk, so this will break things.

Reply:
Done